### PR TITLE
Read subnet proxy configuration from minio or env var

### PR DESCRIPTION
**Pull request by Alevsk**
Screenshot from 2022-02-02 21-55-20

Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Support/Register.tsx [hunk 1]
```diff
@@ -1001,21 +994,33 @@ const Register = ({
                     fontSize: "14px",
                   }}
                 >
-                  For airgap/firewalled environments it is possible to configure
-                  a proxy to connect to SUBNET.
+                  For airgap/firewalled environments it is possible to{" "}
+                  <Link
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Support/Register.tsx:998]: Could we link this to our docs?
`https://docs.min.io/docs/minio-server-configuration-guide.html?ref=con`

Comment (dvaldivia) [reply portal-ui/src/screens/Console/Support/Register.tsx:998]: We should add this new config to our docs @ravindk89 @harshavardhana

Comment (Alevsk) [reply portal-ui/src/screens/Console/Support/Register.tsx:997]: done

**Review by prakashsvmx — APPROVED** [review]
LGTM 👍

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**